src/vm/amd64/asmconstants.h

```diff
@@ -663,7 +663,9 @@ ASMCONSTANTS_C_ASSERT(OFFSETOF__TransitionBlock__m_argumentRegisters == offsetof
 
 #undef ASMCONSTANTS_RUNTIME_ASSERT
 #undef ASMCONSTANTS_C_ASSERT
+#ifndef UNIX_AMD64_ABI
 #undef DBG_FRE
```

Comment:
That is strange, the assembler should get a fully preprocessed file, so it should not encounter the DBG_FRE. Could you please show me the full error message?